platform/alz/alz_policy_default_values.json

```diff
@@ -1,8 +1,45 @@
 {
   "$schema": "https://raw.githubusercontent.com/Azure/Azure-Landing-Zones-Library/main/schemas/default_policy_values.json",
   "defaults": [
+    {
+      "default_name": "private_dns_zone_subscription_id",
+      "description": "The subscription id that hosts the private link DNS zones.",
+      "policy_assignments": [
+        {
+          "parameter_names": [
+            "dnsZoneSubscriptionId"
+          ],
+          "policy_assignment_name": "Deploy-Private-DNS-Zones"
+        }
+      ]
+    },
+    {
+      "default_name": "private_dns_zone_resource_group_name",
+      "description": "The resource group name that hosts the private link DNS zones.",
+      "policy_assignments": [
+        {
+          "parameter_names": [
+            "dnsZoneResourceGroupName"
+          ],
+          "policy_assignment_name": "Deploy-Private-DNS-Zones"
+        }
+      ]
+    },
+    {
+      "default_name": "private_dns_zone_region",
+      "description": "The region short name (e.g. `westus`) that should be used for the region specific private link DNS zones.",
+      "policy_assignments": [
+        {
+          "parameter_names": [
+            "dnsZoneRegion"
+          ],
+          "policy_assignment_name": "Deploy-Private-DNS-Zones"
+        }
+      ]
+    },
     {
       "default_name": "private_dns_zone_acr",
+      "description": "DEPRECATED - please use private_dns_zone_region, private_dns_zone_resource_group_name, and private_dns_zone_subscription_id instead.",
```

Comment:
Why not remove these? We are not using them and the SLZ team are not using them. Highly unlikely a customer is using them for anything production.

The policy is versioned with the library so not impact there either right?

Reply:
Removed